spec/features/post_show_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe 'Posts', type: :system do
+RSpec.describe 'Posts', type: :feature do
   before :all do
     Like.delete_all
     Comment.delete_all
```

Comment:
### Although I am highlighting this file, this suggested change should be done in the `spec/features/posts_index_spec.rb` file. 

- [ ] As highlighted by the previous reviewer, kindly do the following:

- Write a test that redirects to the `Posts#Show` page when you click on a post on the `Posts#Index` page. 

The code you wrote in the screenshot below in the `spec/features/posts_index_spec.rb` file is directing to the `Posts#Show` page from the `Users#Show` page and not the `Posts#Index` page. Kindly write the correct test 👍 :

![image](https://github.com/sheezu900/blog-rails/assets/77106962/f9537cf1-a951-40ee-92f8-4705655aae42)